consensus/babe/types.go

```diff
@@ -18,12 +18,12 @@ package babe
 
 // TODO: change to Schnorrkel keys
 type VrfPublicKey [32]byte
-type VrfPrivateKey [64]byte
+type VrfPrivateKey [64]byte	   
 
 // BabeConfiguration contains the starting data needed for Babe
 // see: https://github.com/paritytech/substrate/blob/426c26b8bddfcdbaf8d29f45b128e0864b57de1c/core/consensus/babe/primitives/src/lib.rs#L132
 type BabeConfiguration struct {
-	SlotDuration       uint64
+	SlotDuration       uint64 // length of a slot in nanoseconds	
```

Comment:
is it nanoseconds or milliseconds? pls provide source

Reply:
based on this its in milliseconds: https://github.com/paritytech/substrate/blob/426c26b8bddfcdbaf8d29f45b128e0864b57de1c/core/consensus/babe/primitives/src/lib.rs#L132

please let me know if that's changed

Reply:
whoops I'll fix that, my bad